.travis.yml

```diff
@@ -16,3 +16,4 @@ install:
 
 script:
     - go test -v ./...
+    - go test -short -bench BenchmarkReference -benchmem
```

Comment:
What is `-short` flag for?

You probably want this line to only run that `BenchmarkReference` benchmark and no tests, right?

If so, I think you should add `-run=^$` or something equivalent.